product-page.css

```diff
@@ -0,0 +1,89 @@
+/* @import url('https://lucidui1.netlify.app/styles.css'); */
+
+body{
+    display:grid;
+    grid-template-columns: 12.5rem 1fr;
+    grid-template-rows: min-content;
+    margin:0;
+}
+
+header {
+    grid-column: 1/3;
+    grid-row:1/1;
+    position: sticky;
+    top:0;
+    z-index: 999;
+  }
+
+  .input{
+      max-width:15rem;
+  }
+
+.sidebar{
+    grid-column:1/2;
+    grid-row:2/-1;
+    height:80vh;
+    position: sticky;
+    top:4rem;
+    overflow-y:scroll;
+    padding:1rem;
+}
+
+.sidebar::-webkit-scrollbar {
+    display: none;
+  }
+
+  main{
+      display:grid;
+      gap:1rem;
+      grid-template-columns: repeat(auto-fit,250px);
+      place-content: center;
+  }
+
+  .main-header{
+      grid-column:1/-1;
+      display:flex;
+      justify-content: center;
+        align-items:center;
+        gap:1rem;
+  }
+
+  .input-with-icon{
+      display:flex;
+      background-color: white;
+      border-radius:1rem;
+      border:1px solid;
+      overflow: hidden;
+      padding:0.3rem;
+  }
+
+  .input-with-icon>input{
+    border: none;
+    width:15rem;
+    padding:0.3rem;
```

Comment:
You can use px for less than 10px instead of rem

Reply:
ok ..thank you for review.